default/buildings.txt

```diff
@@ -231,7 +231,6 @@ BuildingType
                 Planet 
                 Object id = Source.PlanetID
             ]
-            effects = SetTargetConstruction value = Value - 10
```

Comment:
That change only affects a currently commented out building. However, if it were uncommented, I think this will lead to a parse error. So either you leave that commented out building completely unchanged, or you remove the entire EffectsGroup, not just its effects section.